manifests/reposync-crd.yaml

```diff
@@ -1296,6 +1296,26 @@ spec:
                     format: int64
                     minimum: 0
                     type: integer
+                  logLevel:
+                    description: logLevel allows one to override the log verbosity
```

Comment:
`logLevel allows one to override the log verbosity of a reconciler deployment container.`

We need to be explicit that these are for the reconciler deployment containers, not just any old containers.

Reply:
changed